composer.json

```diff
@@ -24,7 +24,8 @@
     "phpunit/phpunit": "^5.7 || ^9.0",
     "php-coveralls/php-coveralls": "^2.1",
     "squizlabs/php_codesniffer": "^3.3",
-    "friendsofphp/php-cs-fixer": "3.2.1 || 2.17.1"
+    "friendsofphp/php-cs-fixer": "3.2.1 || 2.17.1",
+    "phpstan/phpstan": "^1.2"
```

Comment:
Will this cause issues installing the package with dev dependencies on earlier versions if we're saying it's not compatible with them?

Reply:
Yes. I think we should go with a policy of `composer install` is only supported on php 7.3+. I think this policy is fine because the only users of `composer install` are
* CI - which we have already handled with the `composer remove` workaround in this PR.
* maintainers/contributors to stripe-php:
  * I am happy to be restricted to php7.3+ when doing everyday development on stripe-php
  * When I am explicitly trying to test compatibility things with <=php7.2 it will be definitely an inconvenience to not be able to `composer install` without manually working around, and beyond that it will be inconvenient to not be able to utilize `phpstan/php-cs-fixer`, but I think this inconvenience is far outweighed by the additional complexity it would take to continuously maintain compatibility with multiple versions of PHPStan and php-cs-fixer.

Reply:
Sounds good - let's leave a comment explaining this and steps for running <php 7.2 so that this doesn't come back to haunt us down the road once we've forgotten :)